build.sbt

```diff
@@ -203,6 +204,7 @@ lazy val commonSettings = Seq(
     "co.fs2" %% "fs2-core" % V.fs2,
     "org.apache.kafka" % "kafka-clients" % V.kafka,
     "io.confluent" % "kafka-avro-serializer" % V.confluent,
+    "org.apache.avro" % "avro" % V.avro % Compile, // CVE-2023-39410, didn't work as Runtime
```

Comment:
I don't know why `Runtime` doesn't work here.  It didn't work for Guava, either.

Reply:
I do not know why `% Compile` works either, so we are equally ignorant footing 😛

Reply:
Oh, `% Compile` is the default.  It's in the same scope as kafka-avro-serializer, which is what brings it in.  But runtime dependencies should propagate into the runtime of downstream, so I'm confused.  We do similar in lots of other places and it works.